bin/pav.py

```diff
@@ -99,6 +100,21 @@ def record_factory(*fargs, **kwargs):
     result_logger.setLevel(logging.INFO)
     result_logger.addHandler(result_handler)
 
+    # Setup the exception logger.
+    # Exceptions will be logged to this directory, along with other useful info.
+    exc_logger = logging.getLogger('exceptions')
+    exc_handler = RotatingFileHandler(
+        filename=pav_cfg.exception_log.as_posix(),
+        maxBytes=20 * 1024 **2,
```

Comment:
What is the significance of 20 MB?

Reply:
It's a random number.

Reply:
Ok, I guess it's an arbitrary number. I'm really balancing overall file size with not losing useful log information if Pavilion ever puked a crazy amount of log data.